Makefile

```diff
@@ -77,7 +77,7 @@ endif
 
 kafel/include/kafel.h: kafel_init
 kafel/libkafel.a: kafel_init
-	CFLAGS=-fPIE $(MAKE) -C kafel
+	LDFLAGS="" CFLAGS=-fPIE $(MAKE) -C kafel
```

Comment:
Can you add a comment that explains that this is a work-around for Kafel using the parent LDFLAGS incorrectly? So we understand why this is here in 6 months or a year if said issue is fixed. Apart from that LGTM

Reply:
Done.